ruby_test.rb

```diff
@@ -0,0 +1,3 @@
+puts "one"
+puts 'two'
+puts 'foo'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.